dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Your call @lennessyy but this could get very noisy, especially in the JS ecosystem 😅 
My recommendation is to leave it as monthly. 

The other I wonder is if we should add `target-branch` and `commit-message`. That way we could ensure it starts with `chore: `

[Source](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#target-branch).